tasks/scraping.go

```diff
@@ -0,0 +1,47 @@
+package tasks
+
+import (
+	"context"
+
+	"go-google-scraper-challenge/models"
+	"go-google-scraper-challenge/services/scraper"
+
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/beego/beego/v2/task"
+)
+
+type ScrapingTask struct {
+	Tasker *task.Task
+}
+
+func (t *ScrapingTask) Name() string {
```

Comment:
Do we need a function just to get a static value? I think it would be good if we just create a constant variable for it

Reply:
✅  fixed in [f2a2d01](https://github.com/carryall/go-google-scraper-challenge/pull/71/commits/f2a2d01921336072c348d556b81c891efda1160a) also use the function without receiver so we don't have to call them an instance 😄